src/flask_ml/flask_ml_server/models.py

```diff
@@ -170,16 +170,6 @@ class FloatRangeDescriptor(BaseModel):
     max: float
 
 
-class ResponseType(Enum):
-    FILE = 'file'
-    DIRECTORY = 'directory'
-    MARKDOWN = 'markdown'
-    TEXT = 'text'
-    BATCHFILE = 'batchfile'
-    BATCHTEXT = 'batchtext'
-    BATCHDIRECTORY = 'batchdirectory'
-
-
```

Comment:
@atharvakale343 Some backend teams might already be using this, right? This might break their code.

Also, wasn't it better the way it was before? The intention was clearer with "ResponseType.FILE" and the type being "ResponseType" compare to the type just being "Literal['file']".

Reply:
So ResponseType isn’t a user facing type. They never have to use it when defining anything. It is already set as a default value in all models that use ResponseType.

The problem with the earlier approach was that you could set for example a `MarkdownResponse.output_type` to something like `ResponseType.DIRECTORY` which of course does not work. Setting those to literals makes it clear that `output_type` accepts only one value, and that value is already set as a default.